scripts/x-data-grid-pro.exports.json

```diff
@@ -115,18 +119,17 @@
   { "name": "GridColumnMenuState", "kind": "Interface" },
   { "name": "GridColumnOrderChangeParams", "kind": "Interface" },
   { "name": "GridColumnPinningApi", "kind": "Interface" },
-  { "name": "GridColumnPinningMenuItems", "kind": "ExportSpecifier" },
```

Comment:
It needs to be exported or we have a breaking change.

Reply:
Good catch, I'll do a double check on the export files